### ISSUE TITLE
(Accidental major bug) + Fix auto complete in One-word typo

### ISSUE THREAD

**Issue by RaSan147** [Acode-Foundation/Acode#681]
You'll understand by the video.
Acode: heght (missing i), when trying to fix, adding i shows "height", happily clicking that makes it "heightght"

Pydroid: heght, same case, adding i and selecting suggest simply recheck the next characters of the word and if matching fixes as the suggestion says. (Expecting behavior)

https://user-images.githubusercontent.com/34002411/230974457-25978316-870b-4b6d-a6b8-766df017ec6e.mp4

**Comment by RaSan147** [comment]
@deadlyjack  Major issue caught on 4k, look at the title and edit icon

I was editing chat_expression file. But why was Run_me.py showing edit dot beside it!!
Just opened the editor and this happened (using github version of acode)

**Comment by Elovel** [comment]
same problem, full of bug

**Comment by deadlyjack** [comment]
Thanks for reporting, I will fix it.

**Comment by deadlyjack** [comment]
There is no issue like when gboard (google keyboard) is used, only occurs when swift keyboard is used.

**Comment by RaSan147** [comment]
> There is no issue like when gboard (google keyboard) is used, only occurs when swift keyboard is used.

Well the video was recorded using Gboard, let me test with the newest version. Thanks

**Comment by RaSan147** [comment]
Nah, still not working, using 1.8.4 alpha build. Auto-complete keeps the later part as is

**Comment by asl97** [comment]
> There is no issue like when gboard (google keyboard) is used, only occurs when swift keyboard is used.

It is still the same, this issue is with the intergrated completion.

here is a video show casing both the integrated completion and gboard completion

https://github.com/deadlyjack/Acode/assets/4752645/c7f2c14c-8822-4956-8cdb-a0c24387592e

**Comment by deadlyjack** [comment]
SwiftKey keyboard supposed to work like that if you are using SwiftKey keyboard.

**Comment by asl97** [comment]
> SwiftKey keyboard supposed to work like that if you are using SwiftKey keyboard.

do you mean to say acode uses SwiftKey keyboard internally for the suggestion that popup?

I pressed the popup that appears at 2-3 seconds mark.

or do you mean to say this behavior is intended as it is how most keyboard works as they don't look at characters after the cursor for intelligent suggestion like gboard?
